London/8 mahri atdayeva html/css week 3

mahri09 opened the pull request:
<!--

The title for your pull request should be made in this format

CITY CLASS_NO - FIRST_NAME LAST_NAME - MODULE - WEEK_NO

For example,

London Class 7 - Chris Owen - HTML/CSS - Week 1

Please complete the details below this message

-->

**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name: Mahri
- Your City: London
- Your Slack Name: Mahri

# Homework Details

- Module: HTML/CSS 
- Week: 3

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

LucyMac commented on css/order.css:
```diff
@@ -0,0 +1,227 @@
+  width: 130px;
+}
+
+.color {
+  display: flex;
+  text-align: center;
+}
```
Try and give your class names a name that represent what they target or what they do. This class doesn't apply any colour! :)

LucyMac commented on css/order.css:
```diff
@@ -0,0 +1,227 @@
+  border-radius: 4px;
+}
+
+.borders label {
+  display: flex;
+  margin-top: 18px;
+}
```
What does the `.borders` class do? It doesn't seem to be declared anywhere. Perhaps you could rename it and apply it to the `label` elements directly.

LucyMac commented on css/order.css:
```diff
@@ -0,0 +1,227 @@
+input[type="text"],
+input[type="email"],
+select {
+  width: 170px;
```
Avoid setting fixed widths to elements when you know they will need to adapt to different screensizes and therefore should be able to stretch or contract a little to better fill the space.

LucyMac commented on css/order.css:
```diff
@@ -0,0 +1,227 @@
+}
+
+input.address {
+  width: 100%;
```
Something like this is much better!

LucyMac commented on css/order.css:
```diff
@@ -0,0 +1,227 @@
+}
+
+input.postcode {
+  width: 130px;
```
Again, can you use something more fluid such as %?

LucyMac commented on css/style.css:
```diff
@@ -1,9 +1,243 @@
+  background-color: azure;
+}
+
+.products {
+  margin: 4rem 0;
+  text-align: center;
+}
+
+.product-cards {
+  display: flex;
+  justify-content: space-around;
+  margin: 0 5.5rem;
+}
+
+.products h2 {
+  font-family: "Montserrat", sans-serif;
+  font-size: 2.2rem;
+  padding: 1rem;
+}
+
+.products p {
+  font-size: 1.2rem;
+  padding: 1.1rem 0;
+}
+
+.cards {
+  padding: 1.3rem 0;
+}
+
+.card-logos {
```
All really good class names here - makes it instantly understandable which part of the page you're styling ⭐

LucyMac commented on css/style.css:
```diff
@@ -1,9 +1,243 @@
+  padding: 0 5px;
+}
+
+.get-karma {
```
This is a main button which may be re-used elsewhere, so perhaps don't name it by its text content. 
We often refer to buttons as `.button--primary` or `.button--secondary` so we can create the buttons once and then re-use them throughout the site/app (this is following the [BEM notation](https://css-tricks.com/bem-101/) if you're wondering about the double dash)

LucyMac commented on css/style.css:
```diff
@@ -1,9 +1,243 @@
+  width: auto;
+}
+
+.quoto-section .quoto-container {
+  width: 100%;
+  height: 350px;
+  background-color: rgba(224, 99, 58, 0.1);
+  padding: 0 4.5rem;
+}
+
+.quoto-content {
+  margin: auto;
+  padding-top: 2.5rem;
+  display: block;
+}
```
I think you can combine these 2 elements and only use 1 class. See comment in your HTML file.

LucyMac commented on index.html:
```diff
@@ -1,19 +1,147 @@
+          alt="quoto image"
+        />
+        <article class="quoto-container">
+          <div class="quoto-content">
```
What is this `div` doing that `.quoto-container` couldn't do? They are wrapping exactly the same elements. It looks like you could remove the div and just have the article (which by the way could be a `blockquote` element instead)

LucyMac commented on index.html:
```diff
@@ -1,19 +1,147 @@
+              <span>“</span>Wherever I am, I just don’t worry about my
+              connection anymore!<span>”</span>
+            </p>
+            <button class="get-karma">Get Karma today</button>
```
See my CSS comment on not naming things by their text content (this could change and is not re-usable!)

LucyMac commented on index.html:
```diff
@@ -1,19 +1,147 @@
+        </article>
+      </section>
+    </main>
+    <footer>
+      <div class="footer-container">
```
Same thing here - why these 2 elements? They wrap exactly the same elements. Remove the div and move all styles to the `footer`.

LucyMac commented on index.html:
```diff
@@ -1,19 +1,147 @@
+            />
+          </a>
+        </div>
+        <p>@Karma mobility, INC</p>
```
You can replace the @ sign with `&copy;` and it will render as a copyright symbole

LucyMac commented on order.html:
```diff
@@ -0,0 +1,179 @@
+          <h1>Order your Karma wifi device today!</h1>
+          <form action="">
+            <div class="oneline-input">
+              <div class="borders">
```
Could you re-name this something more generic and less tied to its style, for example `input-wrapper`?

LucyMac commented on order.html:
```diff
@@ -0,0 +1,179 @@
+                  required
+                />
+              </div>
+            </div>
```
All the above inputs should be wrapped inside a `<fieldset>` with a `<legend`> to signify that they belong to the same 'topic' i.e. they all refer to the user's contact details. 
The following radio buttons on the other hand deal with details relating to the product that is being bought, so could also have its own `<fieldset>` and `<legend`>

LucyMac reviewed (commented):
Really nice, neat code - a pleasure to read :) Your HTML is very well structured and your CSS classes on the whole well organised - just a few comments on some unnecessary elements and on naming improvements. 
Watch out when giving fixed (pixel) sizes to elements. We often set widths of elements or blocks as percentages or fractions of the space, so that they keep the same relative proportions at different screen sizes (for example 50% will always be half of the space, rather than setting a fixed width which will not adapt to the screen).

On a separate note, I couldn't pull this code down to view the rendered result in my browser. Has this branch been deleted or something? You seem to open a lot of PRs for the same thing so try and keep an eye on this and only open one PR at a time.

mahri09 commented:
Thank you very much Lucy, I really appreciate that you review the codes one by one and make corrections. I will look at them soon. It’s really really helpfull!!!

Sent from my iPhone

> On 29 Nov 2021, at 10:46, Lucy Zidour ***@***.***> wrote:
> 
> ﻿
> @LucyMac commented on this pull request.
> 
> Really nice, neat code - a pleasure to read :) Your HTML is very well structured and your CSS classes on the whole well organised - just a few comments on some unnecessary elements and on naming improvements.
> Watch out when giving fixed (pixel) sizes to elements. We often set widths of elements or blocks as percentages or fractions of the space, so that they keep the same relative proportions at different screen sizes (for example 50% will always be half of the space, rather than setting a fixed width which will not adapt to the screen).
> 
> On a separate note, I couldn't pull this code down to view the rendered result in my browser. Has this branch been deleted or something? You seem to open a lot of PRs for the same thing so try and keep an eye on this and only open one PR at a time.
> 
> In css/order.css:
> 
> > +.color {
> +  display: flex;
> +  text-align: center;
> +}
> Try and give your class names a name that represent what they target or what they do. This class doesn't apply any colour! :)
> 
> In css/order.css:
> 
> > +.borders label {
> +  display: flex;
> +  margin-top: 18px;
> +}
> What does the .borders class do? It doesn't seem to be declared anywhere. Perhaps you could rename it and apply it to the label elements directly.
> 
> In css/order.css:
> 
> > +  display: flex;
> +  margin-top: 18px;
> +}
> +
> +.oneline-input {
> +  display: flex;
> +  justify-content: space-between;
> +}
> +
> +.required {
> +  margin-left: 3px;
> +}
> +input[type="text"],
> +input[type="email"],
> +select {
> +  width: 170px;
> Avoid setting fixed widths to elements when you know they will need to adapt to different screensizes and therefore should be able to stretch or contract a little to better fill the space.
> 
> In css/order.css:
> 
> > +input[type="text"],
> +input[type="email"],
> +select {
> +  width: 170px;
> +  height: 35px;
> +  padding: 3px;
> +  margin: 8px 0;
> +  display: inline-block;
> +  border: 1px solid rgb(104, 101, 101);
> +  border-radius: 4px;
> +  box-sizing: border-box;
> +  vertical-align: middle;
> +}
> +
> +input.address {
> +  width: 100%;
> Something like this is much better!
> 
> In css/order.css:
> 
> > +  box-sizing: border-box;
> +  vertical-align: middle;
> +}
> +
> +input.address {
> +  width: 100%;
> +}
> +
> +select {
> +  width: 210px;
> +  font-style: italic;
> +  color: rgb(110, 107, 107);
> +}
> +
> +input.postcode {
> +  width: 130px;
> Again, can you use something more fluid such as %?
> 
> In css/style.css:
> 
> > +.products {
> +  margin: 4rem 0;
> +  text-align: center;
> +}
> +
> +.product-cards {
> +  display: flex;
> +  justify-content: space-around;
> +  margin: 0 5.5rem;
> +}
> +
> +.products h2 {
> +  font-family: "Montserrat", sans-serif;
> +  font-size: 2.2rem;
> +  padding: 1rem;
> +}
> +
> +.products p {
> +  font-size: 1.2rem;
> +  padding: 1.1rem 0;
> +}
> +
> +.cards {
> +  padding: 1.3rem 0;
> +}
> +
> +.card-logos {
> All really good class names here - makes it instantly understandable which part of the page you're styling ⭐
> 
> In css/style.css:
> 
> > +  margin: auto;
> +  padding-top: 2.5rem;
> +  display: block;
> +}
> +.quoto-content p {
> +  margin-top: 1.8rem;
> +  font-style: italic;
> +  font-size: 2.2rem;
> +}
> +
> +.quoto-content span {
> +  color: #e0633a;
> +  padding: 0 5px;
> +}
> +
> +.get-karma {
> This is a main button which may be re-used elsewhere, so perhaps don't name it by its text content.
> We often refer to buttons as .button--primary or .button--secondary so we can create the buttons once and then re-use them throughout the site/app (this is following the BEM notation if you're wondering about the double dash)
> 
> In css/style.css:
> 
> > +.quoto-section .quoto-container {
> +  width: 100%;
> +  height: 350px;
> +  background-color: rgba(224, 99, 58, 0.1);
> +  padding: 0 4.5rem;
> +}
> +
> +.quoto-content {
> +  margin: auto;
> +  padding-top: 2.5rem;
> +  display: block;
> +}
> I think you can combine these 2 elements and only use 1 class. See comment in your HTML file.
> 
> In index.html:
> 
> > +              class="card-logos"
> +              src="./img/icon-refill.svg"
> +              alt="refill-logo"
> +            />
> +            <p>Pay as you go</p>
> +          </article>
> +        </div>
> +      </section>
> +      <section class="quoto-section">
> +        <img
> +          class="quoto-image"
> +          src="./img/homepage-feature.jpg"
> +          alt="quoto image"
> +        />
> +        <article class="quoto-container">
> +          <div class="quoto-content">
> What is this div doing that .quoto-container couldn't do? They are wrapping exactly the same elements. It looks like you could remove the div and just have the article (which by the way could be a blockquote element instead)
> 
> In index.html:
> 
> > +          </article>
> +        </div>
> +      </section>
> +      <section class="quoto-section">
> +        <img
> +          class="quoto-image"
> +          src="./img/homepage-feature.jpg"
> +          alt="quoto image"
> +        />
> +        <article class="quoto-container">
> +          <div class="quoto-content">
> +            <p>
> +              <span>“</span>Wherever I am, I just don’t worry about my
> +              connection anymore!<span>”</span>
> +            </p>
> +            <button class="get-karma">Get Karma today</button>
> See my CSS comment on not naming things by their text content (this could change and is not re-usable!)
> 
> In index.html:
> 
> > +    <footer>
> +      <div class="footer-container">
> Same thing here - why these 2 elements? They wrap exactly the same elements. Remove the div and move all styles to the footer.
> 
> In index.html:
> 
> > +          <a href="#">
> +            <img
> +              class="footer-icons"
> +              src="./img/facebook-icon.svg"
> +              alt="facebook-icon"
> +            />
> +          </a>
> +          <a href="#">
> +            <img
> +              class="footer-icons"
> +              src="./img/instagram-icon.svg"
> +              alt="instagram-icon"
> +            />
> +          </a>
> +        </div>
> +        ***@***.*** mobility, INC</p>
> You can replace the @ sign with &copy; and it will render as a copyright symbole
> 
> In order.html:
> 
> > +              <a href="#" class="nav-link">Help</a>
> +            </li>
> +            <li class="list-item">
> +              <a href="./order.html" class="nav-link">Login</a>
> +            </li>
> +          </ul>
> +        </nav>
> +      </div>
> +    </header>
> +    <main>
> +      <section class="form-section">
> +        <div class="form-container">
> +          <h1>Order your Karma wifi device today!</h1>
> +          <form action="">
> +            <div class="oneline-input">
> +              <div class="borders">
> Could you re-name this something more generic and less tied to its style, for example input-wrapper?
> 
> In order.html:
> 
> > +                  <option value="texas">Texas</option>
> +                </select>
> +              </div>
> +              <div class="borders">
> +                <label for="postcode"
> +                  >Postcode<span class="required">*</span></label
> +                >
> +                <input
> +                  id="postcode"
> +                  class="postcode"
> +                  type="text"
> +                  name="postcode"
> +                  required
> +                />
> +              </div>
> +            </div>
> All the above inputs should be wrapped inside a <fieldset> with a <legend> to signify that they belong to the same 'topic' i.e. they all refer to the user's contact details.
> The following radio buttons on the other hand deal with details relating to the product that is being bought, so could also have its own <fieldset> and <legend>
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or unsubscribe.
> Triage notifications on the go with GitHub Mobile for iOS or Android.

mahri09 commented:
<!--
/* Font Definitions */
@font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;}
@font-face
	{font-family:"Segoe UI Emoji";
	panose-1:2 11 5 2 4 2 4 2 2 3;}
/* Style Definitions */
p.MsoNormal, li.MsoNormal, div.MsoNormal
	{margin:0cm;
	font-size:11.0pt;
	font-family:"Calibri",sans-serif;}
a:link, span.MsoHyperlink
	{mso-style-priority:99;
	color:blue;
	text-decoration:underline;}
code
	{mso-style-priority:99;
	font-family:"Courier New";}
pre
	{mso-style-priority:99;
	mso-style-link:"HTML Preformatted Char";
	margin:0cm;
	font-size:10.0pt;
	font-family:"Courier New";}
span.HTMLPreformattedChar
	{mso-style-name:"HTML Preformatted Char";
	mso-style-priority:99;
	mso-style-link:"HTML Preformatted";
	font-family:"Courier New";}
.MsoChpDefault
	{mso-style-type:export-only;}
@page WordSection1
	{size:612.0pt 792.0pt;
	margin:72.0pt 72.0pt 72.0pt 72.0pt;}
div.WordSection1
	{page:WordSection1;}
-->Sorry Lucy as I understand you couldn’t open on your browser from my GitHub, what should I do now can you give me suggestion how can I improve it?!Sent from Mail for Windows 10 From: Lucy ZidourSent: Monday, 29 November 2021 10:46 amTo: CodeYourFuture/HTML-CSS-Coursework-Week2Cc: mahri09; AuthorSubject: Re: [CodeYourFuture/HTML-CSS-Coursework-Week2] London/8 mahri atdayeva html/css week 3 (PR ***@***.*** commented on this pull request.Really nice, neat code - a pleasure to read :) Your HTML is very well structured and your CSS classes on the whole well organised - just a few comments on some unnecessary elements and on naming improvements.Watch out when giving fixed (pixel) sizes to elements. We often set widths of elements or blocks as percentages or fractions of the space, so that they keep the same relative proportions at different screen sizes (for example 50% will always be half of the space, rather than setting a fixed width which will not adapt to the screen).On a separate note, I couldn't pull this code down to view the rendered result in my browser. Has this branch been deleted or something? You seem to open a lot of PRs for the same thing so try and keep an eye on this and only open one PR at a time.In css/order.css:> +.color { +  display: flex; +  text-align: center; +} Try and give your class names a name that represent what they target or what they do. This class doesn't apply any colour! :)In css/order.css:> +.borders label { +  display: flex; +  margin-top: 18px; +} What does the .borders class do? It doesn't seem to be declared anywhere. Perhaps you could rename it and apply it to the label elements directly.In css/order.css:> +  display: flex; +  margin-top: 18px; +} + +.oneline-input { +  display: flex; +  justify-content: space-between; +} + +.required { +  margin-left: 3px; +} +input[type="text"], +input[type="email"], +select { +  width: 170px; Avoid setting fixed widths to elements when you know they will need to adapt to different screensizes and therefore should be able to stretch or contract a little to better fill the space.In css/order.css:> +input[type="text"], +input[type="email"], +select { +  width: 170px; +  height: 35px; +  padding: 3px; +  margin: 8px 0; +  display: inline-block; +  border: 1px solid rgb(104, 101, 101); +  border-radius: 4px; +  box-sizing: border-box; +  vertical-align: middle; +} + +input.address { +  width: 100%; Something like this is much better!In css/order.css:> +  box-sizing: border-box; +  vertical-align: middle; +} + +input.address { +  width: 100%; +} + +select { +  width: 210px; +  font-style: italic; +  color: rgb(110, 107, 107); +} + +input.postcode { +  width: 130px; Again, can you use something more fluid such as %?In css/style.css:> +.products { +  margin: 4rem 0; +  text-align: center; +} + +.product-cards { +  display: flex; +  justify-content: space-around; +  margin: 0 5.5rem; +} + +.products h2 { +  font-family: "Montserrat", sans-serif; +  font-size: 2.2rem; +  padding: 1rem; +} + +.products p { +  font-size: 1.2rem; +  padding: 1.1rem 0; +} + +.cards { +  padding: 1.3rem 0; +} + +.card-logos { All really good class names here - makes it instantly understandable which part of the page you're styling ⭐In css/style.css:> +  margin: auto; +  padding-top: 2.5rem; +  display: block; +} +.quoto-content p { +  margin-top: 1.8rem; +  font-style: italic; +  font-size: 2.2rem; +} + +.quoto-content span { +  color: #e0633a; +  padding: 0 5px; +} + +.get-karma { This is a main button which may be re-used elsewhere, so perhaps don't name it by its text content.We often refer to buttons as .button--primary or .button--secondary so we can create the buttons once and then re-use them throughout the site/app (this is following the BEM notation if you're wondering about the double dash)In css/style.css:> +.quoto-section .quoto-container { +  width: 100%; +  height: 350px; +  background-color: rgba(224, 99, 58, 0.1); +  padding: 0 4.5rem; +} + +.quoto-content { +  margin: auto; +  padding-top: 2.5rem; +  display: block; +} I think you can combine these 2 elements and only use 1 class. See comment in your HTML file.In index.html:> +              class="card-logos" +              src="./img/icon-refill.svg" +              alt="refill-logo" +            /> +            <p>Pay as you go</p> +          </article> +        </div> +      </section> +      <section class="quoto-section"> +        <img +          class="quoto-image" +          src="./img/homepage-feature.jpg" +          alt="quoto image" +        /> +        <article class="quoto-container"> +          <div class="quoto-content"> What is this div doing that .quoto-container couldn't do? They are wrapping exactly the same elements. It looks like you could remove the div and just have the article (which by the way could be a blockquote element instead)In index.html:> +          </article> +        </div> +      </section> +      <section class="quoto-section"> +        <img +          class="quoto-image" +          src="./img/homepage-feature.jpg" +          alt="quoto image" +        /> +        <article class="quoto-container"> +          <div class="quoto-content"> +            <p> +              <span>“</span>Wherever I am, I just don’t worry about my +              connection anymore!<span>”</span> +            </p> +            <button class="get-karma">Get Karma today</button> See my CSS comment on not naming things by their text content (this could change and is not re-usable!)In index.html:> +    <footer> +      <div class="footer-container"> Same thing here - why these 2 elements? They wrap exactly the same elements. Remove the div and move all styles to the footer.In index.html:> +          <a href="#"> +            <img +              class="footer-icons" +              src="./img/facebook-icon.svg" +              alt="facebook-icon" +            /> +          </a> +          <a href="#"> +            <img +              class="footer-icons" +              src="./img/instagram-icon.svg" +              alt="instagram-icon" +            /> +          </a> +        </div> +        ***@***.*** mobility, INC</p> You can replace the @ sign with &copy; and it will render as a copyright symboleIn order.html:> +              <a href="#" class="nav-link">Help</a> +            </li> +            <li class="list-item"> +              <a href="./order.html" class="nav-link">Login</a> +            </li> +          </ul> +        </nav> +      </div> +    </header> +    <main> +      <section class="form-section"> +        <div class="form-container"> +          <h1>Order your Karma wifi device today!</h1> +          <form action=""> +            <div class="oneline-input"> +              <div class="borders"> Could you re-name this something more generic and less tied to its style, for example input-wrapper?In order.html:> +                  <option value="texas">Texas</option> +                </select> +              </div> +              <div class="borders"> +                <label for="postcode" +                  >Postcode<span class="required">*</span></label +                > +                <input +                  id="postcode" +                  class="postcode" +                  type="text" +                  name="postcode" +                  required +                /> +              </div> +            </div> All the above inputs should be wrapped inside a <fieldset> with a <legend> to signify that they belong to the same 'topic' i.e. they all refer to the user's contact details.The following radio buttons on the other hand deal with details relating to the product that is being bought, so could also have its own <fieldset> and <legend>—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.  

LucyMac commented:
What's going on her @mahri09? Can you remove that huge bit of code posted as a comment above?

mahri09 commented:
> @LucyMac I  didn't see that sorry

mahri09 commented:
one question @LucyMac, after your review, when I worked on my project and pushed it pushed another branch of the same name. Is it possible in github or maybe I messed up before 😥

LucyMac commented:
Yes I think it's something to do with that. But it's weird because you
shouldn't be able to have 2 branches with the same name 🤔

Go to GitHub desktop, make sure you're on one of the 2 branches, and rename
it - even just one character each a word on the end for example.

Then see if your PRs update with the new name - they may not because the
'remote' name won't have changed. You'll then need to look up in Google
"GitHub change remote branch name"



On Tue, 30 Nov 2021 at 18:37, mahri09 ***@***.***> wrote:

> one question @LucyMac <https://github.com/LucyMac>, after your review,
> when I worked on my project and pushed it pushed another branch of the same
> name. Is it possible in github or maybe I messed up before 😥
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week2/pull/347#issuecomment-982910558>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACETDDYTHKBMOM6LJGOBLKLUOUKVNANCNFSM5I3F67JA>
> .
>
-- 

*Lucy Zidour*


*+44 748 2172 789*

mahri09 commented:
> 

@LucyMac yeah there are two branches on my repository. I am using the Github desktop application first time I did a mistake before I was using terminal or git bash I think better go back to my terminals :) Now this is the latest branch I pushed new codes https://github.com/mahri09/HTML-CSS-Coursework-Week2/tree/London/8-Mahri-Atdayeva-HTML/CSS-Week-3-last-updated can I leave it and work on it after!? I also send pull request from this branch to CYF

mahri09 commented:
@LucyMac I found the problem when I rename my branch from the browser it automatically creates new branch and leave another one too and changes my working branch for the new one. just now I renamed one of the two branches to make it different, when I open my repo I found out there 3 branches 😥. before I renamed my branch name to correct it manually that's why it is duplicated

mahri09 commented:
so the solution I can go and change my branch to this reviewed branch,  copy and paste my new codes into this branch cause they are on my new branches and push it to this branch and delete other branches to make clear here

LucyMac commented:
Basically:

- you must not have duplicate branch names
- make sure all your latest code is on *one* branch
- open a PR for that branch and that branch only
- close any other PRs




On Tue, 30 Nov 2021 at 20:24, mahri09 ***@***.***> wrote:

> so the solution I can go and change my branch to this reviewed branch,
> copy and paste my new codes into this branch cause they are on my new
> branches and push it to this branch and delete other branches to make clear
> here
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/CodeYourFuture/HTML-CSS-Coursework-Week2/pull/347#issuecomment-982990821>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACETDD4J2FBIZUCRK3X4RM3UOUXHLANCNFSM5I3F67JA>
> .
>
-- 

*Lucy Zidour*


*+44 748 2172 789*

mahri09 commented:
thank you @LucyMac I pushed all my code on this branch and delete the other once

mairacagri reviewed (commented):
Great job Mahri. i learn lot from your code especially how to give a right class names. Very clean looking code. Also great use of comments, i will apply the same technic for my homework this week!

mahri09 commented:
> Great job Mahri. i learn lot from your code especially how to give a right class names. Very clean looking code. Also great use of comments, i will apply the same technic for my homework this week!

Thank you Maira! I believe You will do much better day by day!
